PostAnalyzer/functions/plotSelectionEfficiencies.py

```diff
@@ -0,0 +1,100 @@
+import os 
+import ROOT
+import sys
+import numpy as np
+from ModuleRunnerBase import GenericPath
+
+
+def main():
+    generic = GenericPath()
+
+    step_efficiencies = ['weights','triggers','nominal']
+    BinNumber = len(step_efficiencies)
+    years = ['UL18']
```

Comment:
if it's a list, then the path should change accordingly. 
Otherwise you need just a string

Reply:
The path is now updated at the beginning of the loop